pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -139,25 +136,13 @@ public String getControllerVipUrl() {
     return _controllerConf.generateVipUrl();
   }
 
-  public static synchronized void create(PinotHelixResourceManager helixResourceManager, ControllerConf controllerConf,
+  public PinotLLCRealtimeSegmentManager(PinotHelixResourceManager helixResourceManager, ControllerConf controllerConf,
```

Comment:
I think you can remove the old constructor and only keep this one. (For the test, use mock)

Reply:
Done.